src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/coordinator/ManagedIndexCoordinatorIT.kt

```diff
@@ -125,7 +125,7 @@ class ManagedIndexCoordinatorIT : IndexStateManagementRestTestCase() {
             assertPredicatesOnMetaData(
                 listOf(index to listOf(ManagedIndexSettings.POLICY_ID.key to fun(policyID: Any?): Boolean = policyID == null)),
                 getExplainMap(index),
-                true
```

Comment:
Just curious why the change was needed

Reply:
emmm, not sure anymore, I change it back to true, and it can still pass